### PR TITLE
Changes due to having a new home page

### PR DESCRIPTION
**Pull request by qbzzt**
1.  Added a tutorials page 
2. Moved the FAQs to link to the homepage
3. Removed no longer needed pages with app lists
4. Made the home page redirect a copy of `docs/README.md`.
5. Added a link to the homepage.

### CODE REVIEW

File: src/.vuepress/config.js [hunk 1]
```diff
@@ -173,14 +141,7 @@ module.exports = {
           collapsable: false,
           sidebarDepth: 0,
         }
-      ],
-      '/faqs/': [
-        {
-          children: [ '' ],
-          collapsable: false,
-          sidebarDepth: 1,
```
Comment (elenadimitrova) [inline src/.vuepress/config.js:181]: Also delete the `/faqs` folder and subpages

File: src/.vuepress/components/Home.vue [hunk 2]
```diff
@@ -14,6 +14,10 @@
         Optimistic Ethereum
       </h1>
 
+      <script>
+        location.href="/docs"
```
Comment (elenadimitrova) [inline src/.vuepress/components/Home.vue:18]: This seems hacky, isn't there a cleaner way to set the homepage to /docs? Will also allow us to remove the content of this <template>

File: src/.vuepress/config.js (hunk 1 above)
Comment (qbzzt) [reply src/.vuepress/config.js:181]: Done

File: src/.vuepress/components/Home.vue (hunk 2 above)
Comment (qbzzt) [reply src/.vuepress/components/Home.vue]: Fixed.

**Review by K-Ho — APPROVED** [review]

**Comment by platocrat** [comment]
LF
LFGPEP
G

**Pull request merged**